src/components/ConditionalFilter/API/initialState/orders/useInitialOrderState.ts

```diff
@@ -25,6 +25,14 @@ const getCustomer = (customer: string[]) => {
   return "";
 };
 
+const getIDs = (ids: string[]) =>
+  ids.map(id => ({
+    type: "ids",
+    label: id,
+    value: id,
+    slug: id,
+  }));
```

Comment:
suggestion: you are not getting value but map over, I'd recommend name it something like this `mapIdsToOption`